content/03.ml-concepts.md

```diff
@@ -1,16 +1,16 @@
-## Tip 1: Concepts that apply to machine learning also apply to deep learning {#concepts}
+## Tip 1: Successful deep learning applications require best practices for machine learning and model evaluation {#concepts}
 
-While deep learning has inarguably surpassed general machine learning for artificial intelligence and predictive modeling applications in terms of public awareness, deep learning still remains a subfield of machine learning.
-DL has proven to be an extremely powerful paradigm capable of outperforming "traditional" or "conventional" machine learning approaches. 
-This is partly owed to the fact that DL algorithms are less dependent on manual feature extraction and are capable of achieving high predictive performance when operating on raw-data, such as images and text. directly.
-In contrast, traditional machine learning approaches often require manual feature extraction to produce acceptable results.
-However, the majority of best practices for machine learning still apply to deep learning.
-Like all computational methods, experiments involving deep learning should be conducted in a systematic manner and accompanied by reproducible instructions as well as rigorous model evaluation.
 
-The ability of deep learning to perform implicit feature extraction can be regarded as a double-edged sword.
+In recent years, deep learning has proven to be an extremely powerful paradigm capable of outperforming "traditional" or "conventional" machine learning approaches.
+The recent success behind applying deep learning is partly owed to the fact that DL algorithms are less dependent on manual feature extraction and are capable of achieving high predictive performance when operating on raw-data, such as images and text directly.
+In contrast, traditional machine learning approaches often require manual feature extraction to produce acceptable results. 
+However, the ability of deep learning to perform implicit feature extraction can be regarded as a double-edged sword.
 For instance, those developing deep learning models not only need to ensure that the data contains information relevant to the problem at hand but is also free of systematic biases.
 As deep learning models are capable of extracting and learning from complex patterns in a training dataset, DL models are also susceptible to systematic noise and non-salient information, which can not only hamper performance but also lead to spurious conclusions.
-Biases in testing data can also unduly influence measures of model performance, and it may be difficult to identify confounders from the model directly.
+
+Since deep learning still remains a subfield of machine learning, the majority of best practices for machine learning still apply to deep learning.
```

Comment:
I think that `still remains a subfield` sounds really wordy and kind of odd here. I would change it to `Since deep learning is still a subfield`.

Reply:
totally agree!